epaint/src/tessellator.rs

```diff
@@ -705,6 +725,74 @@ impl Tessellator {
         }
         assert_eq!(chars.next(), None);
     }
+    
+    pub fn tessellate_multicolor_text(
```

Comment:
I rather have just one `tesselate_text`. With `TextColorMap::default()` they should act the same anyway.